server/seeders/20201109034858-organizations-seed.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
         `
         insert into organization (organizationname, organizationtypeenum, organizationstate, organizationstateuniqueid, recordcreatetimestamp, recordcreateuser_uuid, recordupdatetimestamp, recordupdateuser_uuid) select 'San Francisco Fire Department','EMS',  '06','S38-50827',now(),user_uuid, now(), user_uuid from batsuser where email = 'batsadmin@c4sf.me';
         insert into organization (organizationname, organizationtypeenum, organizationstate, organizationstateuniqueid, recordcreatetimestamp, recordcreateuser_uuid, recordupdatetimestamp, recordupdateuser_uuid) select 'King American','EMS', '06','S3850502',now(),user_uuid, now(), user_uuid from batsuser where email = 'batsadmin@c4sf.me';
-        insert into organization (organizationname, organizationtypeenum, organizationstate, organizationstateuniqueid, recordcreatetimestamp, recordcreateuser_uuid, recordupdatetimestamp, recordupdateuser_uuid) select 'American Medical Response (AMR)','EMS', '06','S39-50088',now(),user_uuid, now(), user_uuid from batsuser where email = 'batsadmin@c4sf.me';
+        insert into organization (organizationname, organizationtypeenum, organizationstate, organizationstateuniqueid, recordcreatetimestamp, recordcreateuser_uuid, recordupdatetimestamp, recordupdateuser_uuid, ismfaenabled) select 'American Medical Response (AMR)','EMS', '06','S39-50088',now(),user_uuid, now(), user_uuid, true from batsuser where email = 'batsadmin@c4sf.me';
```

Comment:
Now that 2FA is disabled by default, I am re-enabling it on the AMR organization so that we can test 2FA